composer.json

```diff
@@ -45,7 +46,7 @@
         "phpunit/phpunit": "^9.5",
         "phpunitgoodpractices/polyfill": "^1.6",
         "phpunitgoodpractices/traits": "^1.9.2",
-        "symfony/phpunit-bridge": "^6.0",
+        "symfony/phpunit-bridge": "^6.2.3@dev",
```

Comment:
As soon as 6.2.3 is released, we can revert this package back to stable. See symfony/symfony#48718

Reply:
Symfony 6.2.3 has been released. I'm reverting the `@dev`.